app/src/main/res/layout/fragment_game.xml

```diff
@@ -1,20 +1,32 @@
 <?xml version="1.0" encoding="utf-8"?>
-<com.daprlabs.aaron.swipedeck.layouts.SwipeFrameLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:swipedeck="http://schemas.android.com/apk/res-auto"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:id="@+id/swipeLayout"
-    android:orientation="vertical">
+<RelativeLayout android:layout_height="match_parent" android:layout_width="match_parent"
+    xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <com.daprlabs.aaron.swipedeck.SwipeDeck
-        android:id="@+id/cardStack"
+    <TextView
+        android:id="@+id/tvTimer"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textSize="50sp"
+        android:gravity="center_vertical|center_horizontal"
```

Comment:
can combine both and make it "center". center_vertical is to specify the we need to center it vertically and center_horizontal is to specify that we need to center it horizontally. center is to center both vertically and horizontally